cachi2/core/package_managers/bundler/main.py

```diff
@@ -24,5 +31,49 @@ def fetch_bundler_source(request: Request) -> RequestOutput:
 
 def _resolve_bundler_package(package_dir: RootedPath, output_dir: RootedPath) -> list[Component]:
     """Process a request for a single bundler package."""
-    parse_lockfile(package_dir)
+    dependencies = parse_lockfile(package_dir)
+    _get_package_metadata(package_dir, dependencies)
     return []
+
+
+def _get_package_metadata(
```

Comment:
Is this really metadata? It seems to be just package name and optional version.

Reply:
Yeah, the function name is ported from cachito. How about `_get_package_info` ?

Reply:
Well, we also use the term `metadata` in the [pip](https://github.com/containerbuildsystem/cachi2/blob/main/cachi2/core/package_managers/pip.py#L276) module. Would something like `_get_package_info` be more appropriate?

Reply:
Yes, I'd say `info` sounds better. `metadata` is data about data, this looks like just regular data.

Reply:
Isn't it at the end just the SBOM component?

Reply:
It is, but I think where the data came from matters. I think it's the difference between a "main package/project/app" and its dependencies... in the end, all of them can be considered "packages". But I still think the distiction is useful. Some random ideas for renaming it:

- _get_root_package_info
- _get_app_info
- _get_main_package_info

The first suggestion does not have any precedent in Cachi2, the second is used in some variable names, the third is still what we mostly use.

Or just keep it as `_get_package_info` and clarify it in the docstring, as Alexey suggested.

Reply:
gomod: `main_module`
npm: `get_main_package`
pip: `get_pip_metadata`
:exploding_head:

Reply:
> _get_app_info

Not a bad name :).

Reply:
How about `_get_main_package_name_and_version` ?
It is more verbose, but it is clearer than _info_.